build.gradle

```diff
@@ -89,7 +89,7 @@ subprojects {
 
                 // TODO: Unreleased dependencies.
                 // These must already be installed in the local maven repository.
-                netty: 'io.netty:netty-codec-http2:4.1.0.Beta5-SNAPSHOT',
+                netty: 'io.netty:netty-codec-http2:4.1.0.Beta6-SNAPSHOT',
```

Comment:
I'm thinking for non-required changes we wait until the next beta release. For important changes, then yes, we use unreleased version of Netty and we delay our release until Netty does theirs.